partiql-eval/src/eval.rs

```diff
@@ -397,12 +400,11 @@ impl Evaluable for EvalUnpivot {
             out.push(Value::Tuple(Box::new(t)));
         }
 
-        self.output = Some(Value::Bag(Box::new(Bag::from(out))));
-        self.output.clone()
```

Comment:
Just realized `self.output` wasn't being used at all for all of these nodes. Nice that we can remove the clones.